packages/alpha/CHANGELOG.md

```diff
@@ -3,82 +3,121 @@
-* **vite:** adding common js as fallback [release] ([#1187](https://github.com/cognitedata/cognite-sdk-js/issues/1187)) ([85564e1](https://github.com/cognitedata/cognite-sdk-js/commit/85564e1430e0f721117269b7223b82ccabdf98eb))
+* use enum for connector status ([#1175](https://github.com/cognitedata/cognite-sdk-js/issues/1175)) ([ad0d6d9](https://github.com/cognitedata/cognite-sdk-js/commit/ad0d6d9c3243cd1121d1c02f462e552107319fd4))
 
 
 
 
 
-# [1.0.0-rc.4](https://github.com/cognitedata/cognite-sdk-js/compare/@cognite/sdk-alpha@1.0.0-rc.3...@cognite/sdk-alpha@1.0.0-rc.4) (2025-01-08)
+# [0.33.0](https://github.com/cognitedata/cognite-sdk-js/compare/@cognite/sdk-alpha@0.32.2...@cognite/sdk-alpha@0.33.0) (2024-11-21)
 
-**Note:** Version bump only for package @cognite/sdk-alpha
 
+### Features
+
+* **alpha:** multiple updates for simint APIs ([#1174](https://github.com/cognitedata/cognite-sdk-js/issues/1174)) ([96bddf1](https://github.com/cognitedata/cognite-sdk-js/commit/96bddf183eb46c3edb97be793edb8e8e8e5e857f))
 
 
 
 
-# [1.0.0-rc.3](https://github.com/cognitedata/cognite-sdk-js/compare/@cognite/sdk-alpha@1.0.0-rc.1...@cognite/sdk-alpha@1.0.0-rc.3) (2024-11-07)
+
+## [0.32.2](https://github.com/cognitedata/cognite-sdk-js/compare/@cognite/sdk-alpha@0.32.1...@cognite/sdk-alpha@0.32.2) (2024-11-07)
 
 **Note:** Version bump only for package @cognite/sdk-alpha
 
 
 
 
 
-# [1.0.0-rc.1](https://github.com/cognitedata/cognite-sdk-js/compare/@cognite/sdk-alpha@1.0.0-rc.0...@cognite/sdk-alpha@1.0.0-rc.1) (2024-09-18)
+## [0.32.1](https://github.com/cognitedata/cognite-sdk-js/compare/@cognite/sdk-alpha@0.32.0...@cognite/sdk-alpha@0.32.1) (2024-10-07)
 
 **Note:** Version bump only for package @cognite/sdk-alpha
 
 
 
 
 
-# [1.0.0-rc.0](https://github.com/cognitedata/cognite-sdk-js/compare/@cognite/sdk-alpha@0.30.1...@cognite/sdk-alpha@1.0.0-rc.0) (2024-08-26)
+# [0.32.0](https://github.com/cognitedata/cognite-sdk-js/compare/@cognite/sdk-alpha@0.31.3...@cognite/sdk-alpha@0.32.0) (2024-09-17)
+
+
+### Features
+
+* **simint:** add and remove properties from simulators API ([#1147](https://github.com/cognitedata/cognite-sdk-js/issues/1147)) ([4cd6467](https://github.com/cognitedata/cognite-sdk-js/commit/4cd6467658fde491c20f274fbd8fa6af55610c60))
+
+
+
+
+
+## [0.31.3](https://github.com/cognitedata/cognite-sdk-js/compare/@cognite/sdk-alpha@0.31.1...@cognite/sdk-alpha@0.31.3) (2024-08-26)
+
+
+### Bug Fixes
+
+* dummy release to override a bad one ([#1139](https://github.com/cognitedata/cognite-sdk-js/issues/1139)) ([5b036da](https://github.com/cognitedata/cognite-sdk-js/commit/5b036dabd4630b45d51558ee7f95d951c7227137))
 
 
-### chore
 
-* upgrade to typescript 5 ([#1135](https://github.com/cognitedata/cognite-sdk-js/issues/1135)) ([1530809](https://github.com/cognitedata/cognite-sdk-js/commit/15308098e64ec27a6c33049b2712f9cbc49dac38))
 
 
-### BREAKING CHANGES
```

Comment:
feels like changelog needs maybe a manual merge, dont know how important it is ? for now ?

Reply:
I hope the changelog will be fixed when we merge release-v10 into master